external-crates/move/move-compiler/tests/move_check/parser/constant_type_annotation_invalid.exp

```diff
@@ -0,0 +1,6 @@
+error[E01013]: Requires a type annotation 'name: type ='
+  ┌─ tests/move_check/parser/constant_type_annotation_invalid.move:2:11
+  │
+2 │     const EThreadMaxxed = 1;
+  │           ^^^^^^^^^^^^^ Add type annotation to this constant
```

Comment:
I would probably switch this to pointing to the `=` and saying `"Expected a type annotation for this constant, e.g. 'EThreadMaxxed: <type>'"` or something